tests/rkt_api_service_test.go

```diff
@@ -323,23 +333,62 @@ func NewAPIServiceListInspectPodsTest() testutils.Test {
 		if err != nil {
 			t.Fatalf("Cannot generate types.Hash from %v: %v", imageHash, err)
 		}
-		pm := schema.BlankPodManifest()
-		pm.Apps = []schema.RuntimeApp{
+
+		podManifests := []struct {
+			mfst  schema.PodManifest
+			net   string
+			state int
+		}{
 			{
-				Name: types.ACName("rkt-inspect"),
-				Image: schema.RuntimeImage{
-					Name: types.MustACIdentifier("coreos.com/rkt-inspect"),
-					ID:   *imgID,
+				// 1, Good pod.
+				schema.PodManifest{
+					ACKind:    schema.PodManifestKind,
+					ACVersion: schema.AppContainerVersion,
+					Apps: []schema.RuntimeApp{
+						{
+							Name: types.ACName("rkt-inspect"),
+							Image: schema.RuntimeImage{
+								Name: types.MustACIdentifier("coreos.com/rkt-inspect"),
+								ID:   *imgID,
+							},
+							Annotations: []types.Annotation{{Name: types.ACIdentifier("app-test"), Value: "app-test"}},
+						},
+					},
+					Annotations: []types.Annotation{
+						{Name: types.ACIdentifier("test"), Value: "test"},
+					},
 				},
-				Annotations: []types.Annotation{{Name: types.ACIdentifier("app-test"), Value: "app-test"}},
+				"default",
+				0,
+			},
+			{
+				// 2, Bad pod, won't be launched correctly.
```

Comment:
You could rename the network below from "foo" to "non-existent-network" so it is more obvious why it is a bad pod.